app/boards/shields/pico87/pico87.zmk.yml

```diff
@@ -0,0 +1,8 @@
+file_format: "1"
+id: pico87
+name: Pico 87
+type: shield
+url: https://github.com/mountainmannm/Bolt-Industries/
+requires: [rpi_pico]
```

Comment:
We currently don't have metadata for the `rpi_pico` board that would have `exposes: [rpi_pico]` that this would be compatible with. Not sure what's the exact solution here, do we add a `app/boards/arm/rpi_pico/rpi_pico.zmk.yml`? cc: @petejohanson

Reply:
> We currently don't have metadata for the `rpi_pico` board that would have `exposes: [rpi_pico]` that this would be compatible with. Not sure what's the exact solution here, do we add a `app/boards/arm/rpi_pico/rpi_pico.zmk.yml`? cc: @petejohanson

Yes, we probably do need to add metadata file there. I need to double check, the *exposes* was intended for the generic "interconnect" concept. Originally the `depends` was consider to work with either a board ID or an interconnect ID, I'm not sure we fully fleshed that out or not though.

Ideally we should be able to just depend on the *board ID* to start, and not create some interconnect until we actually have/need it. No one has *yet* created enough RPi Pico drop in replacements it's necessary at this point.

Reply:
I added `app/boards/arm/rpi_pico/rpi_pico.zmk.yml`, which basically looks like `app/boards/arm/adafruit_kb2040/adafruit_kb2040.zmk.yml`.